rpcs3/Emu/Cell/lv2/sys_net/lv2_socket_p2p.cpp

```diff
@@ -367,7 +370,9 @@ s32 lv2_socket_p2p::poll(sys_net_pollfd& sn_pfd, [[maybe_unused]] pollfd& native
 	// Check if it's a bound P2P socket
 	if ((sn_pfd.events & SYS_NET_POLLIN) && !data.empty())
 	{
-		sys_net.trace("[P2P] p2p_data for vport %d contains %d elements", vport, data.size());
+		if (data.size())
```

Comment:
This scope can only be entered if data is not empty anyway, so this is a redundant check
also, why the sudden change of checking size instead of empty?